docs/content/quickstart.md

```diff
@@ -4,51 +4,51 @@ description: "start here!"
 type: docs
 ---
 
-# installing
+# Installing
 
-download the installer or the zip file from the [Releases](https://github.com/workspacer/workspacer/releases/latest) page and install/unzip it anywhere you like. 
+Download the installer or the zip file from the [Releases](https://github.com/workspacer/workspacer/releases/latest) page and install/unzip it anywhere you like. 
 
-alternatively, workspacer is available via [Chocolatey](https://community.chocolatey.org/packages/workspacer):
+Alternatively, workspacer is available via [Chocolatey](https://community.chocolatey.org/packages/workspacer):
 
 ```
 choco install workspacer
 ```
 
-and [Scoop](https://scoop.sh/):
+and via [Scoop](https://scoop.sh/):
 
 ```
 scoop bucket add extras
 scoop install workspacer
 ```
 
-workspacer automatically checks for updates. however, if you are using a package manager with either Chocolatey or Scoop, you should uncomment/add this line in your workspacer config to disable autoupdates:
+Workspacer automatically checks for updates. However, if you are using a package manager (either Chocolatey or Scoop), you should uncomment/add this line in your workspacer configuration file to disable autoupdates:
 
 ```cs
 context.Branch = Branch.None;
 ```
 
-# configuring
+# Configuring
 
-by default, workspacer loads a default configuration, but you will likely want to customize this in order to do things like modify the widgets on the bar, add workspaces, switch workspace containers, or otherwise change the default behavior of workspacer.
+Workspacer loads a default configuration with almost everything you'd need, but you will likely want to modify things like [key bindings](#how-do-i-register-custom-keybindings), [widgets](#menu-bar-widgets) on the bar, [workspaces](#how-do-i-define-custom-workspaces), [layouts](#layout-engines), [plugins](#plugins) or otherwise the default behavior of workspacer.
```

Comment:
How can I reference this headers that are in other pages? (If there's any way to do it without typing the full url)

Reply:
#/config#how-do-i-register-custom-keybindings maybe?

Reply:
Yup that worked